src/main/webapp/style.css

```diff
@@ -131,23 +144,34 @@ body {
   overflow: hidden;
 }
 
+.replies {
+  border-left: #ddd solid 5px;
+  margin-left: 8px;
+  padding-left: 8px;
+}
+
 #page {
   display: flex;
   justify-content: center;
 }
 
 #new-comment-row {
-  height: 200px;
-  max-height: 200px;
+  height: 150px;
+  max-height: 150px;
   overflow: hidden;
 }
 
-#new-comment-form {
+#new-comment-form,
+#reply-form {
   width: 100%;
 }
 
+#reply-form {
+  margin-top: 20px;
+}
+
 #discussion-comments {
```

Comment:
Let's order discussion-comments properly. Even if we want to have "#" together. (Spoke to Sahil a bit about this file).

Reply:
Done!